configs/apache_camel.json

```
@@ -1,20 +1,58 @@
 {
   "index_name": "apache_camel",
-  "stop_urls": [],
+  "only_content_level": true,
+  "stop_urls": [
+    "https://camel.apache.org/categories/",
+    "https://camel.apache.org/blog/page/",
+    "index.html"
+  ],
+  "start_urls": [
+    { "url": "https://camel.apache.org/blog/", "selectors_key": "blog" },
+    { "url": "https://camel.apache.org/camel-k/", "selectors_key": "antora-pages" },
+    { "url": "https://camel.apache.org/camel-quarkus/", "selectors_key": "antora-pages" },
+    { "url": "https://camel.apache.org/camel-kafka-connector/", "selectors_key": "antora-pages" },
+    { "url": "https://camel.apache.org/camel-spring-boot/", "selectors_key": "antora-pages" },
+    { "url": "https://camel.apache.org/camel-karaf/", "selectors_key": "antora-pages" },
+    { "url": "https://camel.apache.org/components/", "selectors_key": "antora-pages" },
+    { "url": "https://camel.apache.org/manual/", "selectors_key": "antora-pages" },
+    "https://camel.apache.org/download/",
+    "https://camel.apache.org/community/books",
+    "https://camel.apache.org/community/articles",
+    "https://camel.apache.org/docs/"
+  ],
   "sitemap_urls": [
     "https://camel.apache.org/sitemap.xml"
   ],
   "selectors": {
-    "lvl0": "article h1",
-    "lvl1": "article h2",
-    "lvl2": "article h3",
-    "lvl3": "article h4",
-    "lvl4": "article h5",
-    "lvl5": "article h6",
-    "text": "article p, article li"
+    "default": {
+      "lvl0": { "selector": "", "default_value": null },
```

```suggestion
      "lvl0": { "selector": "", "default_value": "" },
```
default_value can not be `null`